source/topics/troubleshooting/view-open-support-tickets.md

```diff
@@ -0,0 +1,5 @@
+Sign in/up at [https://aptible.zendesk.com/hc/en-us/requests](https://aptible.zendesk.com/hc/en-us/requests). You can also request permission to view all of your organization's tickets by [sending us a note](https://support.aptible.com/contact/).
+
+If you submitted a ticket via [support@aptible.com](support@aptible.com), follow the link at the bottom of the email to see all open tickets ("_You can view this and your other tickets on the web_")
+
+NB: If you are submitting a ticket for the first time, you'll need to create a password in order to login to the zendesk account we've automatically generated for you.
```

Comment:
s/zendesk/Zendesk/ and consider replacing "NB: If" with "Note that if". We haven't adopted a language style guide so I'm fine leaving NB as is.